Airports/EGCC/SMR/Regions.txt

```diff
@@ -109,7 +113,8 @@ N053.20.39.462 W002.16.54.370
 N053.20.48.213 W002.16.36.440
 ;outer_taxiways
```

Comment:
Can we add a line break before each new polygon perhaps? Just to separate out each section?

Reply:
I will be doing that shortly